webserver/README.md

```diff
@@ -0,0 +1,25 @@
+#### Service Webserver for Elasticsearch
+---
+For this service we are using the latest version of Nginx
+
+```Dockerfile
+FROM nginx:latest
+```
+
+The **nginx.conf** is being filled with a basic configuration where the proxy_pass of the web server root path `/` is serving the Elasticsearch service deployed in the same stack. For this purpose, two new ENV vars were used to **nginx.conf** used to whitelist an IP and to specify the name of the elasticsearch service.
```

Comment:
break the line to improve readability
change `two new ENV vars were used to` to `two new ENV vars were used in`